### DISCUSSION TITLE
Result Set Size Estimation for Tag-Based Queries

### DISCUSSION THREAD

**Discussion by silvanheller** [vitrivr/cineast#180] [Ideas]
During discussions, the idea came up that we could estimate the result set size for tag-based queries by precomputing the number of items per tag and then providing that number to the user at query-time so the selectivity of a query can be estimated. I'll leave this idea here for reference.

**Comment by lucaro** [comment]
Sure, you can do that for single tags, but without knowing the ngrams, any estimation for multiple tags can be very rough.
